WeCantSpell.Hunspell/SpellCheckResult.cs

```diff
@@ -9,6 +9,15 @@ public readonly struct SpellCheckResult
 
     public static SpellCheckResult DefaultWrong { get; } = new SpellCheckResult(root: null, SpellCheckResultType.None, correct: false);
 
+    internal static SpellCheckResult Fail(string? root, SpellCheckResultType info) =>
```

Comment:
These should be used more and may reduce code size in some of the huge methods in this code base.